apps/wallet/src/ui/app/pages/initialize/backup/index.tsx

```diff
@@ -54,55 +53,127 @@ const BackupPage = ({ mode = 'created' }: BackupPageProps) => {
-                            {mnemonic ? (
-                                <div className={st.mnemonic}>
-                                    {mnemonic}
-                                    <CopyToClipboard
-                                        txt={mnemonic}
-                                        className={st.copy}
-                                        mode="plain"
-                                    >
-                                        COPY
-                                    </CopyToClipboard>
-                                </div>
-                            ) : (
-                                <Alert>{error}</Alert>
-                            )}
-                        </Loading>
-                        <div className={st.info}>
-                            Your recovery phrase makes it easy to back up and
-                            restore your account.
-                        </div>
-                        <div className={st.info}>
-                            <div className={st.infoCaption}>WARNING</div>
-                            Never disclose your secret recovery phrase. Anyone
-                            with the passphrase can take over your account
-                            forever.
+            <div className="flex flex-col flex-nowrap items-center justify-center">
+                <div className="p-7.5 pt-10 rounded-20 bg-alice-blue shadow-wallet-content flex flex-col flex-nowrap items-center justify-center w-popup-width h-popup-height">
+                    <div className="text-center flex gap-1.25 flex-col justify-center items-center">
+                        <div className="rounded-full w-12 h-12 border-dotted border-success border-2 flex items-center justify-center mb-2.5 p-1">
+                            <div className="bg-success rounded-full h-8 w-8 flex items-center justify-center">
+                                <Icon
+                                    icon={SuiIcons.ThumbsUp}
+                                    className="text-white text-2xl"
+                                />
+                            </div>
                         </div>
-                    </>
-                ) : null}
-                <div className={st.fill} />
-                <Button
-                    type="button"
-                    className={st.btn}
-                    size="large"
-                    mode="primary"
-                    onClick={() => navigate('/')}
-                >
-                    Open Sui Wallet
-                    <Icon icon={SuiIcons.ArrowLeft} className={st.arrowUp} />
-                </Button>
-            </CardLayout>
+                        <Heading
+                            variant="heading1"
+                            color="gray-90"
+                            as="h1"
+                            weight="bold"
+                            leading="none"
+                        >
+                            {`Wallet ${
```

Comment:
Branch all of the copy, not just the copy subset.

Reply:
done